File: webowe2tiGim/WebPageWithModel/WebPageWithModel/Models/Person.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace WebPageWithModel.Models {
    public class Person {
        [Display(Name = "Imię")]
        public string FirstName { get; set; }
        [Display(Name = "Nazwisko")]
        public string LastName { get; set; }
        [Display(Name = "Wiek")]
        public int Age { get; set; }
        [Display(Name = "Wykształcenie")]
        public Eduction MyEduction { get; set; }
    }

    public enum Eduction {
        Podstawowe = 1,
        Średnie = 2,
        Wyższe = 3,
        Brak = 4
    }
}

File: js/js2/Person.js
class Person{
  constructor(name,lastName,age){
    this.name = name;
    this.lastName = lastName;
    this.age = age;
  }
  info(){
    return `Informacja o osobie: ${this.name} ${this.lastName} wiek: ${this.age}`;
  }
}
export default Person;